### PR TITLE
Tooltip action

### PR DESCRIPTION
**Pull request by endigo9740**
See the ticket for details.

### CODE REVIEW

**Comment by endigo9740** [comment]
This is ready to test. Feedback is encouraged!

File: src/lib/utilities/Tooltip/tooltip.ts [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+		elemContainer.appendChild(node);
+	};
+	createElemContainer();
+
+	// Create the tooltip element
+	// prettier-ignore
+	const createElemTooltip = (): void => {
+		elemTooltip = document.createElement('div');
+		elemTooltip.classList.add('tooltip', `tooltip-${position}`, 'hidden', regionTooltip);
+        if (args.background) { elemTooltip.classList.add(args.background); }
```
[the remark this answers is not in the document]
Comment (endigo9740) [reply src/lib/utilities/Tooltip/tooltip.ts:48]: Yeah I'm aware, I typically opt for that when the response is a return statement:

`if (condition) return;`

I've been writing these this way for 20 years though, so may take some effort to break that habit.

**Comment by Sarenor** [comment]
Still need to remove
```ts
// Tooltips
export { default as Tooltip } from './utilities/Tooltip/Tooltip.svelte';
```
from ``src\lib\index.ts``

File: src/routes/(inner)/utilities/tooltips/+page.svelte [hunk 2]
```diff
@@ -0,0 +1,129 @@
+<script lang="ts">
+	// Components
+	import DataTable from '$lib/components/Table/DataTable.svelte';
+	// Utilities
+	import CodeBlock from '$lib/utilities/CodeBlock/CodeBlock.svelte';
+	import { tooltip } from '$lib/utilities/Tooltip/tooltip';
+
+	// Examples
+	const examples: any = {
```
[the remark this answers is not in the document]
Comment (endigo9740) [reply src/routes/(inner)/utilities/tooltips/+page.svelte:9]: Typescript is throwing red warning for `examples.top` and similar if I remove this. Given it's small use case I might keep this in place for now.

[the remark this answers is not in the document]
Comment (endigo9740) [reply src/routes/(inner)/utilities/tooltips/+page.svelte:9]: Exported the type from the action file and adjusted each example to use this. Seems `padding` was not set as optional, so fixed that while I was at it!

**Comment by endigo9740** [comment]
@Sarenor This appears to have been done already. Make sure you're on the appropriate branch perhaps?

@ryceg All items resolved, feel free to recheck. However, I've noticed this started happening in our `index.ts` file recently. Since you have a good understanding of Typescript, any suggestions here? This is making it a bit noisy and hard to find legit errors.

Screen Shot 2022-09-30 at 10 58 26 AM

**Pull request merged**